lib/puppet/type/scheduled_task.rb

```diff
@@ -102,6 +102,9 @@ def insync?(current)
           * `start_date` ---  The date when the trigger should first become active.
             Defaults to the current date. You should format dates as YYYY-MM-DD,
             although other date formats may work. (Under the hood, this uses `Date.parse`.)
+			`minutes_interval` --- The repeat interval in minutes.
```

Comment:
Looks like we have a mix of tabs and spaces here.  We use spaces for indentation in the Puppet codebase.